src/main/java/com/prgrms/modi/error/ErrorResponse.java

```diff
@@ -0,0 +1,29 @@
+package com.prgrms.modi.error;
+
+import org.springframework.http.HttpStatus;
+
+/**
+ * Created : Rosa
+ */
+public class ErrorResponse {
+
+    private final String errorMessage;
+    private final String requestUri;
+    private final HttpStatus status;
+
+    public ErrorResponse(String errorMessage, String requestUri, HttpStatus status) {
+        this.errorMessage = errorMessage;
+        this.requestUri = requestUri;
+        this.status = status;
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("ErrorResponse{");
+        sb.append("errorMessage='").append(errorMessage).append('\'');
+        sb.append(", requestUri='").append(requestUri).append('\'');
+        sb.append(", status=").append(status);
+        sb.append('}');
+        return sb.toString();
+    }
```

Comment:
* 최근에 Apache Commons Lang 3 라이브러리의 여러 기능들을 써보면서 깔끔하고 유용하다는 생각이 들었어요. 이번 프로젝트에 적용해보는건 어떨까요?

> 참고 [예시 링크](https://recordsoflife.tistory.com/474), [공식 문서](https://commons.apache.org/proper/commons-lang/javadocs/api-release/index.html)

* 의존성 추가

```groovy
implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.11'
```

* 적용한 코드
```suggestion
    @Override
    public String toString() {
        return new ToStringBuilder(this, ToStringStyle.SHORT_PREFIX_STYLE)
            .append("errorMessage", errorMessage)
            .append("requestUri", requestUri)
            .append("status", status)
            .toString();
    }
```

Reply:
잭슨의 강의에 나왔던 부분이네요. 저도 보면서 간결해보여 좋다고 생각했습니다. 별도 의존성 추가 없이도 되던데, Spring Starter 어딘가에 의존성이 이미 추가되어 있는걸까요?